### PR TITLE
Don't throw an error if no task stack  on pass.

### PR DESCRIPTION
**Pull request by Psykar**

### CODE REVIEW

**Comment by Psykar** [comment]
@dtkav I've got the code ready to return the satellite task stack instead if that makes more sense. (I'm leaning that way)

File: missioncontrol/v0/pass_task_stack.py [hunk 1]
```diff
@@ -4,7 +4,12 @@
 
 
 def get_task_stack(uuid):
-    return  Pass.objects.get(uuid=uuid).task_stack.to_dict()
+    try:
+        return  Pass.objects.get(uuid=uuid).task_stack.to_dict()
```
Comment (dtkav) [inline missioncontrol/v0/pass_task_stack.py]: I think you can restructure this so it returns an "ObjectDoesNotExist" error, which already has an error handler.
Something like:
```
TaskStack.get(pass__uuid=uuid).to_dict()
```

**Comment by dtkav** [comment]
The issue with returning the default task stack is that a client won't know if it has been overridden or not.
We could add a read-only field to indicate that?

**Comment by Psykar** [comment]
> The issue with returning the default task stack is that a client won't know if it has been overridden or not.

My thinking couldn't actually come up with a case for that being a problem, given you can't update an existing task stack anyway. But fair comment.

> We could add a read-only field to indicate that?
Feels hackish to me, and wouldn't make sense for that field to be on the /passes/{uuid}/task-stack/ url, but would instead be on the /passes/{uuid}/ - so this fetch wouldn't know anyway.

I'll just have it None for now, and client can fetch it off the satellite if so.

**Review by dtkav — APPROVED** [review]
+1

**Pull request merged**